pybotvac/robot.py

```diff
@@ -132,7 +132,7 @@ def start_cleaning(self, mode=2, navigation_mode=1, category=None, boundary_id=N
 
         return response
 
-    def start_spot_cleaning(self, spot_width=400, spot_height=400):
+    def start_spot_cleaning(self, mode=2, modifier=2, spot_width=400, spot_height=400):
```

Comment:
```suggestion
    def start_spot_cleaning(self, spot_width=400, spot_height=400, mode=2, modifier=2):
```

Put the additional parameters at the end so that positional calls do not fail.

Reply:
Agree!